### PR TITLE
adding wk8 schedule

### PR DESCRIPTION
**Pull request by laurafitzgerald**
@JameelB 
adding the schedule for tomorrow. Using this as a template for discussion in the meeting.

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@JameelB @damienomurchu updated the schedule. Added in a Rebase Race in the afternoon session so we can really bed in that knowledge. Maybe we can all participate in that. :)

File: topic02-schedule/book-schedule/img/04.December.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Schedule for December
+
+## Week 8: Website Review
```
Comment (damienomurchu) [inline topic02-schedule/book-schedule/img/04.December.md]: Just need to change this to week 11, other than that content looks good. 

File needs to be moved up a directory to student-help-guide-content/topic02-schedule/book-schedule/04.December.md by the way (it's currently in the img directory)

**Review by damienomurchu — DISMISSED** [review]
content good, just 1/ 2 small changes needed

File: topic02-schedule/book-schedule/img/04.December.md (hunk 1 above)
Comment (JameelB) [reply topic02-schedule/book-schedule/img/04.December.md]: also need some line breaks to have each time in one line. Need 2 white spaces at the end of each sentence

Comment (laurafitzgerald) [reply topic02-schedule/book-schedule/img/04.December.md]: thanks guys will fix.

**Comment by laurafitzgerald** [comment]
@JameelB @damienomurchu pushed changes.

**Review by JameelB — APPROVED** [review]
Looks great! Rendering correctly with tutors as well

**Comment by laurafitzgerald** [comment]
@JameelB thanks for the verification. merging

**Pull request merged**